src/redux/books/books.js

```diff
@@ -1,27 +1,25 @@
 const ADD_BOOK = 'ADD_BOOK';
 const REMOVE_BOOK = 'REMOVE_BOOK';
 
```

Comment:
- [ ] Let's implement the duck's pattern correctly.

  ```
  const ADD_BOOK = 'bookstore/books/ADD_BOOK'
  const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK'
  ```

Reply:
Thank you @julie-ify , I have made the correction